spec/danger/helpers/comment_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "danger/helpers/comment"
+
+describe Danger::Comment do
+  describe ".from_github" do
+    it "initializes with GitHub comment data structure" do
+      github_comment = { id: 42, body: "github comment" }
+
+      result = described_class.from_github(github_comment)
+
+      expect(result).to have_attributes(id: 42, body: "github comment")
+    end
+  end
+
+  describe ".from_gitlab" do
+    it "initializes with Gitlab comment data structure" do
+      GitlabComment = Struct.new(:id, :body)
+      gitlab_comment = GitlabComment.new(42, "gitlab comment")
+
+      result = described_class.from_gitlab(gitlab_comment)
+
+      expect(result).to have_attributes(id: 42, body: "gitlab comment")
+    end
+  end
+
+  describe "#generated_by_danger?" do
+    it "returns true when body contains generated_by_{identifier}" do
+      comment = described_class.new(42, "generated_by_orta")
```

Comment:
I personally avoid use `let` because it is easier to move code / delete them without having leftovers, it is self-contained. [source](https://robots.thoughtbot.com/lets-not)